src/marqo/README.md

```diff
@@ -2,7 +2,19 @@
 Thank you for contributing to Marqo! Contributions from the open source community help make Marqo be the tensor engine
 you want. 
 
-### A. Run the Marqo application locally (outside of docker):
+Before starting, clone the github repo
```

Comment:
This may not be applicable in all options. Maybe we should move this into Option A